src/main/java/life/qbic/portal/portlet/OMEROClientPortlet.java

```diff
@@ -78,27 +76,20 @@
     private static final Logger LOG = LogManager.getLogger(OMEROClientPortlet.class);
 
     ///////////////////////////////
-
-    private HashMap<String, Long> revProjMap = new HashMap<String, Long>();
-    private HashMap<String, Long> revDsMap;
```

Comment:
Did you talk to @luiskuhn about removing this maps?

Reply:
they are not used in the code since I refactored it in the port to vaadin 8.

Reply:
I see, thanks for the clarification